azext_iot/central/_help.py

```diff
@@ -253,6 +254,70 @@ def _load_central_users_help():
     """
 
 
+def _load_central_api_token_help():
+    helps[
+        "iot central app api-token"
+    ] = """
+        type: group
+        short-summary: Create and Manage API tokens .
+    """
+
+    helps[
+        "iot central app api-token create"
+    ] = """
+        type: command
+        short-summary: Create a new API token in the application
```

Comment:
nit:

```
long-summary: The only time you will see the value of this token is when creating the token. Ensure you store this token somewhere securely, as if you lose it, you will need to create another.
```

Reply:
Agree on adding additional info like this, long-summary is a good place. These are the kind of additional infos that really help user understanding and may reduce support tickets/issues down the line.